src/docs/projects/staticanalysis-bot.rst

```diff
@@ -64,105 +59,99 @@ Just replace ``LOGIN`` with your username (e.g. ``michel``); ``api-YOURTOKEN`` w
+Once created, please add the required scope to read your secret: ``secrets:get:repo:github.com/mozilla-releng/services:branch:master`` (or ``secrets:get:garbage/michel/staticanalysis-bot-dev`` if you're using your own secret)
 
-Add the Taskcluster scope needed to read the secret previously mentioned: ``secrets:get:repo:github.com/mozilla-releng/services:branch:master`` (or ``secrets:get:garbage/michel/staticanalysis-bot-dev`` if you're using your own secret)
+.. note:: 
 
-You may also want to add the scope needed to send you an email report: ``notify:email:michel@email.com`` (if your email is ``michel@email.com``)
+  Your newly created credentials are stored in ``~/.config/please/config.toml``
 
-To summarize, you need to setup your client (if your login is ``michel``), like this:
 
-============= ====================================================================
-Key           Value
-============= ====================================================================
-ClientId      ``mozilla-auth0/ad|Mozilla-LDAP|michel/static-analysis-dev``
-Description   My own static analysis dev. client
-Client Scopes ``secrets:get:repo:github.com/mozilla-releng/services:branch:master``
-              ``notify:email:michel@email.com``
-============= ====================================================================
+3. Develop with Docker
+""""""""""""""""""""""
 
+Run the following (where ``XXX`` is the Taskcluster access token):
 
-.. warning::
+.. code-block:: shell
 
-  Save the **access token** provided by Taskcluster after creating your client, it won't be displayed afterwards
+  ./please shell staticanalysis/bot
 
+Once the initial build finishes, you should get a green Nix shell, running in ``/app/src/staticanalysis/bot``.
 
-3. Project shell
-""""""""""""""""
+.. code-block:: shell
 
-Run the following (where ``XXX`` is the Taskcluster access token):
+  export TRY_TASK_ID="xxx"
+  export TRY_GROUP_ID="yyy"
+  export TASKCLUSTER_SECRET="path/to/your/secret"
+  static-analysis-bot
 
-.. code-block:: shell
+4. Develop without Docker
+"""""""""""""""""""""""""
 
-  ./please shell staticanalysis/bot \
-    --taskcluster-client-id="mozilla-auth0/ad|Mozilla-LDAP|michel/static-analysis-dev" \
-    --taskcluster-access-token=XXX
+It's now possible to develop the project without using Docker.
 
-Once the initial build finishes, you should get a green Nix shell, running in ``/app/src/staticanalysis/bot``.
+.. warning::
 
+  This is pretty experimental at this point, and you will encounter some pain points around Taskcluster integration.
 
-4. Setup a Phabricator test
-"""""""""""""""""""""""""""
 
+First, you need to create Python 3 virtual environment, and setup the project and its dependencies there:
 
-.. note::
-  Make sure your Taskcluster secret has a ``phabricator`` reporter setup, as follows (with a valid Phabricator uri & token):
+.. code-block:: shell
 
-  .. code-block:: yaml
+  mkvirtualenv -p /usr/bin/python3 sa-bot
```

Comment:
Only On MacOS, we presume that `virtualenvwrapper` will be installed in `/usr/local/virtualenvwrapper.sh` as a default instalation based on `python3` installed from `brew`. 
```
pip install virtualenvwrapper
source /usr/local/bin/virtualenvwrapper.sh
```